### PR TITLE
[MIG][10.0] partner_contact_nutrition

### PR DESCRIPTION
**Pull request by lasley**
* Rename manifest
* Bump version
* Change openerp references to odoo

Depends:
- [x] OCA/product-attribute#213

### CODE REVIEW

File: partner_contact_nutrition/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Ursa Information Systems <http://ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Partner Contact Nutrition Information",
+    "summary": "Provide caloric intake",
+    "version": "10.0.1.0.1",
```
Comment (max3903) [inline partner_contact_nutrition/__manifest__.py]: `10.0.1.0.0`

**Review by max3903 — CHANGES_REQUESTED** [review]

**Comment by rafaelbn** [comment]
@lasley please could you rebase! we would like to review! Thanks!

**Comment by rafaelbn** [comment]
@max3903 @lasley we are in the OCA code sprint, if you help us we can review all the partner_contact_nutrition modules. Let us know! Tomorrow last day!

**Review by aming27 — COMMENTED** [review]
I can't reproduce the tab in contact view!

**Comment by lasley** [comment]
Version fixed and dependency for `product_uom` added. This should work now

**Comment by yajo** [comment]
Travis is :x:, not sure why though; could you check?

**Comment by rafaelbn** [comment]
Hi @lasley , tested. This attributes


 are only for contacts, not for companies, please change this. Thanks!

File: partner_contact_nutrition/__manifest__.py (hunk 1 above)
Comment (rafaelbn) [reply partner_contact_nutrition/__manifest__.py]: @max3903 attended in https://github.com/OCA/partner-contact/pull/349/commits/e6b7fc4c2214038bef8bc969ac28f59047f6f481

**Comment by yajo** [comment]
You can use https://github.com/OCA/partner-contact/tree/10.0/partner_contact_personal_information_page to fix what @rafaelbn said.

**Review by gmeficent — COMMENTED** [review]
Hello Lasley,

I don't know the original purpose of the module, but It seems out of context to me. Considering the technical language used, the diet forms part of a diagnostic and a date should be ascribed to it so attaching it to the contact info when its quite a transient thing seems weird.

Following the same reasoning I would ascribe it only to patients not the whole res.partner record set or maybe to a patient diagnostic.

I'm working with vertical-medical with @jbeficent and @rgarnau, if we can help with anything let us know.
Regards

**Comment by rafaelbn** [comment]
Please read related issue ROADMAP: https://github.com/OCA/partner-contact/issues/346

**Comment by rafaelbn** [comment]
Sorry the main issue is: https://github.com/OCA/partner-contact/issues/335

**Comment by lasley** [comment]
There is also my proposal for a Vertical Health repo in the [mailing list](https://www.odoo-community.org/groups/contributors-15/contributors-48397?mode=date&date_begin=2016-12-01&date_end=2017-01-01), in which I elaborate on why I do not believe these modules should be in Vertical Medical.

It should be noted that nutrition information can be used outside of the medical context. Examples include:
* Astronauts - physical condition is an HR issue
* Sky Diving - Weight and Height information are required for both the customers and employees in order to judge chute size
* Personal Health - Diabetes and Fitness tracking using IoT devices (FitBit, Glucose Monitors, etc)

None of the above would be applicable to a patient, which is the context used for a third party recipient of medical care.

**Comment by gmeficent** [comment]
Yes, I though so afterwards. Thanks for the elaboration.

**Comment by yajo** [comment]
@lasley any comments on https://github.com/OCA/partner-contact/pull/349#issuecomment-305130285?

**Comment by lasley** [comment]
@Yajo - I don't see why using that page would be bad, although it could possibly clunk that page up a bit. 

@max3903, thoughts on https://github.com/OCA/partner-contact/pull/349#issuecomment-305130285?

**Comment by max3903** [comment]
@lasley Putting the nutrition info on the personal info tab? Seems like a good idea.

My customer is on version 9.0 and we are not talking about migrating to version 10 yet. So go ahead and we'll adapt. Thanks.

**Comment by yajo** [comment]
Yeah, you'll see it's a very dummy module, but solves one simple need for many addons in this repo: display some personal info only for contacts. 😊

File: partner_contact_nutrition/views/res_partner_view.xml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                        </div>
+                        <label for="protein_intake"/>
+                        <div>
+                            <field name='protein_intake' class="oe_inline"/>
+                            <field name='protein_intake_uom'
+                                   class="oe_inline"
+                                   placeholder="UoM"/>
+                        </div>
+                    </group>
+                    <group name="other_info" string="Other info"/>
```
Comment (yajo) [inline partner_contact_nutrition/views/res_partner_view.xml:41]: Why this?

**Comment by rafaelbn** [comment]
Hi @max3903 , it looks you are blocking this PR with you https://github.com/OCA/partner-contact/pull/349#pullrequestreview-11397956 😄 . Please could you update you review? Thanks!

**Review by max3903 — APPROVED** [review]

**Comment by max3903** [comment]
@rafaelbn Sorry. Thanks for the ping.

**Review by rafaelbn — APPROVED** [review]
Tested functionally in runbot 👍

**Pull request merged**